ansible/osp_cleanup.yaml

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ansible-playbook
+---
+- hosts: localhost
+  vars_files: "vars/default.yaml"
+
+  roles:
+  - oc_local
+
+  tasks:
+    - name: Cleanup of OpenStack
+      environment:
+        PATH: "{{ oc_env_path }}"
+        KUBECONFIG: "{{ kubeconfig }}"
+      block:
+        - name: switch to openstack project/namespace
+          shell: >
+            oc project openstack
+
+        - name: delete openstack.org CRD's
+          shell: >
+            oc get route -n openshift-image-registry -o json | jq ".items[0].spec.host" -r
```

Comment:
This doesn't quite look right.

I usually do with something like this:

oc get crd | grep openstack | cut -d " " -f 1 | xargs oc delete crd

Reply:
yea copy/paste error I think. I'll fix it.